test/google.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { browser } from 'protractor';
+
+describe('Given a SDET learning protractor', () => {
+  describe('when open Google Page', () => {
+    beforeEach(async () => {
+      await browser.get('http://www.google.com');
+    });
+
+    it('then should have a title', async () => {
```

Comment:
esa descripcion no dice nada, sería mejor algo como `Then the page should have a title`